query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/update/UpdateStart.java

```diff
@@ -41,4 +41,32 @@
    */
   @NonNull
   UpdateStart usingTimestamp(@NonNull BindMarker bindMarker);
+
+  /**
+   * Adds a {@code USING TTL} clause to this statement with a literal value. Setting a value of
+   * {@code null} will remove the {@code USING TTL} clause on this statement. Setting a value of
+   * {@code 0} will update the data and remove any TTL on the column when the statement is executed,
+   * overriding any TTL (table or column) that may exist in Cassandra.
+   *
+   * <p>If this method or {@link #usingTtlInSeconds(BindMarker) } is called multiple times, the
+   * value from the last invocation is used.
+   *
+   * @param ttlInSeconds Time, in seconds, the inserted data should live before expiring.
+   */
+  @NonNull
+  UpdateStart usingTtlInSeconds(int ttlInSeconds);
+
+  /**
+   * Adds a {@code USING TTL} clause to this statement with a bind marker. Setting a value of {@code
+   * null} will remove the {@code USING TTL} clause on this statement. Binding a value of {@code 0}
+   * will update the data and remove any TTL on the column when the statement is executed,
+   * overriding any TTL (table or column) that may exist in Cassandra.
+   *
+   * <p>If this method or {@link #usingTtlInSeconds(int)} is called multiple times, the value from
+   * the last invocation is used.
+   *
+   * @param bindMarker A bind marker that is understood to be a value in seconds.
+   */
+  @NonNull
+  UpdateStart usingTtlInSeconds(@NonNull BindMarker bindMarker);
```

Comment:
I think the feedback was to rename the parameter, but I'm not sure about the method itself. I think it's good to keep it close to the CQL clause.
cc @adutra

Reply:
Yes, all I wanted was to see a mention somewhere that the bind marker is meant to be in seconds. The old method name was better imo.

Reply:
I'll revert the method name change